eng/scripts/Copy-ApiVersion.ps1

```diff
@@ -1,77 +1,138 @@
 [CmdletBinding()]
 param (
-    [Parameter(Mandatory = $true)]
+    [Parameter(Mandatory)]
```

Comment:
This syntax change is relatively new. I don't recommend using it in case someone is using an older PowerShell version.